bokeh/server/server.py

```diff
@@ -169,7 +177,7 @@ def stop(self, wait: bool = True) -> None:
         self._tornado.stop(wait)
         self._http.stop()
 
-    def unlisten(self) -> None:
+    def unlisten(self) -> Generator[Coroutine[Any, Any, None], None, None]:
```

Comment:
Why does this code still use `yield`? Perhaps this could be turned into an `async` function?

Reply:
yes this should be awaited https://www.tornadoweb.org/en/stable/httpserver.html#tornado.httpserver.HTTPServer.close_all_connections